test/src/config/constants.ts

```diff
@@ -12,6 +12,8 @@ export const urls = {
 };
 
 export const MILLISECONDS_IN_ONE_MINUTE = 60000;
+export const MILLISECONDS_IN_ONE_DAY = MILLISECONDS_IN_ONE_MINUTE * 60 * 24;
```

Comment:
I see you're already defining this in `src/identity-utils.ts` unless I'm mistaken.

Reply:
i took a stab at trying to make things a little more consistent.  but i'm going to make a ticket about trying to more globally replace all of our instances of doing manual math like 60 * 24 * 1000, etc etc and i'll add the ticket as a todo